src/foam/comics/v2/DAOBrowseControllerView.js

```diff
@@ -152,18 +152,19 @@ foam.CLASS({
                 .start(self.Cols)
                   .start()
                     .addClass(self.myClass('browse-title'))
-                    .translate(menuId + ".title", config$browseTitle)
+                    .translate(menuId + ".browseTitle", config$browseTitle)
                   .end()
                   .startContext({ data: self }).tag(self.CREATE).endContext()
                   .callIf(config$primaryAction, function() {
                     this.startContext({ data: self }).tag(config$primaryAction, { size: 'LARGE' }).endContext();
                   })
                 .end()
-                .callIf(config$browseSubtitle.length > 0, function() {
+                .callIf((menuId + ".browseSubtitle").length > 0, function() {
                   this
                     .start()
                       .addClass(self.myClass('browse-subtitle'))
-                      .translate(menuId + ".subTitle", config$browseSubtitle)
+                      .translate(self.cls_.id + '.VIEW_ALL', self.VIEW_ALL)
```

Comment:
I thought we decided to not add View All here so that users could customize their sub-title to anything? FYI: @nanoscott

Reply:
I've updated to unify each menu's title/subTitle and look fine. But @nanoscott Could you tell me which menu you are on it please?

Reply:
@kgrgreer @nanoscott I will remove 'View All' from subTitle to customize